### DISCUSSION TITLE
Editing Date problem

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#4060] [Help]
Hello, so I've been working on my form and I noticed that I cannot edit the date. When I tried to edit the edit, it will always go back to it's original date. I compared it to my newly made app (which updating its date is perfectly working), I see no difference in its functionalities in the settings. What seems to be wrong?

**Comment by mjashanks** [comment]
@lykaodan  - at which point does it go back to its original date? After you save & re-open the record?

If you check the `Data` section in Budibase, what does the date show? Can you give a few screenshots? Thanks!

**Comment by lykaodan** [comment]
Hello,

So the problem is, I inputted the data and the date will be January 20,
2022. But, when I save it, it is displayed as January 19. It turns out
the date will be saved backwards.

**Reply by mjashanks** [reply]
Thanks, looks like a bug. 

Just to confirm, when you say "..when I save it, it is displayed as January 19" - where is it displaying?

I can't seem to reproduce the issue. My Steps
1. Create new record, set date including time
2. Save
3. In list screen, date displayed correctly
4. Edit record
5. Date displayed correctly
6. Update time
7. Date displayed correctly

I have tried this with a different timezone, as I think that this is related.

Could you give some steps to reproduce, and some screenshots of where the date/time is displaying incorrectly? Also - are you using Budibase's internal tables? or SQL?

Thanks!
